### PR TITLE
feature 1400

### PR DESCRIPTION
**Pull request by brouillette**
Explorer -> Variant search: Assembly combobox now only displays choices for which data exist inside of Gohan

### CODE REVIEW

File: src/components/discovery/VariantSearchHeader.js [hunk 1]
```diff
@@ -2,8 +2,12 @@ import React, { useState } from "react";
 import { Form, Select } from "antd";
 import PropTypes from "prop-types";
 import LocusSearch from "./LocusSearch";
+import { useSelector } from "react-redux";
 
 const VariantSearchHeader = ({dataType, addVariantSearchValues}) => {
+    const varOvRes = useSelector((state) => state.explorer.variantsOverviewResponse);
+    const ovAsmIds = varOvRes?.assemblyIDs !== undefined ? Object.keys(varOvRes?.assemblyIDs) : [];
```
Comment (gsfk) [inline src/components/discovery/VariantSearchHeader.js:9]: variable names here are obscure to me

**Review by gsfk — APPROVED** [review]
Looks good! A nice-to-have feature here would be that, in cases where there is only one assembly, it is already selected. This saves the user a pointless click, and gives more information than an empty box. Could always be another ticket I suppose. 

I also quibbled with some variable names, but feel free to take that with a grain of salt.

File: src/components/discovery/VariantSearchHeader.js (hunk 1 above)
Comment (brouillette) [reply src/components/discovery/VariantSearchHeader.js:9]: I agree
The linter doesn't like lines longer than 120 characters, so I abbreviated them

File: src/components/discovery/VariantSearchHeader.js [hunk 2]
```diff
@@ -17,18 +21,6 @@ const VariantSearchHeader = ({dataType, addVariantSearchValues}) => {
...
-    // files ingested with GRCh37 reference need to be searched using assemblyId "Other"
-
-    // so if assembly is "Other", pass that value to the form, but use
-    // "GRCh37" as the reference for gene lookup in Gohan
-        if (value === "Other") {
-            setLookupAssemblyId("GRCh37");
-            return;
-        }
-
-        setLookupAssemblyId(value);
```
Comment (gsfk) [inline src/components/discovery/VariantSearchHeader.js:31]: setLookupAssemblyId(value) needs to stay or variant search will break. 

only  the` if (value === "Other")` block needs to go.

Comment (brouillette) [reply src/components/discovery/VariantSearchHeader.js:31]: fixed

File: src/components/discovery/VariantSearchHeader.js (hunk 1 above)
Comment (brouillette) [reply src/components/discovery/VariantSearchHeader.js:9]: see https://github.com/bento-platform/bento_web/pull/199/commits/d70960701bfb1b8a67ef59bbfe7d0c3f336471cc for the nice-to-have!

**Pull request merged**